shopelectro/management/commands/test_db.py

```diff
@@ -118,11 +121,20 @@ def get_name(number, parent=None):
     def create_products(self, categories, tags):
         def create_images(page: Page):
             def create_image(file_path, slug):
-                Image.objects.create(
+                # product "/catalog/products/2/" contains image
+                image = Image.objects.create(
                     model=page,
                     slug=slug,
                     image=ImageFile(open(file_path, mode='rb'))
                 )
+                # save files to media folder
+                file_src = open(file_path, mode='rb')
```

Comment:
Open a file with `with-statement` to release it after use
```python
with open(file_path, mode='rb') as file:
    image = Image.objects.create(
        model=page,
        slug=slug,
        image=ImageFile(file)
    )
    ...
```

Reply:
TODO